docs/5.0/quickstart.md

```diff
@@ -6,7 +6,7 @@ description: The quick start guide for how to set up modern SSH access to cloud
 # Teleport Quick Start
 
 This tutorial will guide you through the steps needed to install and run
-Teleport on a single node, which could be your local machine but we recommend a
+Teleport Unified Access Plane, which could be your local machine but we recommend a
```

Comment:
why do we recommend a vm?

Reply:
For the quickstart we decided to write it for Junior Sysadmins and SREs, from the work for Teleport User Profiles. I've removed the VM reference to simply make the setup about running Teleport on Linux Machines. 

> This tutorial will guide you through the steps needed to install and run Teleport on Linux machine(s).

In future vresions we'll also have a quickstart focused on SREs using a Kubernetes cluster.